src/main/java/lottoauto/domain/lotto/Lotto.java

```diff
@@ -0,0 +1,40 @@
+package lottoauto.domain.lotto;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Lotto {
+
+    private final Set<LottoNumber> lottoNumbers;
+
+    public Lotto(List<Integer> lottoNumbers) {
```

Comment:
Set 타입으로 로또를 생성할 수 있는 생성자를 추가하면 어떨까요?
```suggestion
    public Lotto(Set<Integer> lottoNumbers) {
```